internal/controller/dscinitialization/dscinitialization_test.go

```diff
@@ -50,6 +50,18 @@ var _ = Describe("DataScienceCluster initialization", func() {
 
 		AfterEach(cleanupResources)
 
+		It("Should not have finalizer on DSCI resource", func(ctx context.Context) {
+			// then
+			foundDsci := &dsciv1.DSCInitialization{}
+			Eventually(objectExists(applicationName, applicationNamespace, foundDsci)).
+				WithContext(ctx).
+				Should(BeTrue())
+			Eventually(func() bool {
```

Comment:
is not clear to me if finalizers were present in the first place in the resources set up in this test.